src/components/ebay-combobox/component.js

```diff
@@ -15,11 +15,19 @@ export default {
     },
 
     isExpanded() {
-        return this.expander.expanded;
+        return this.expander && this.expander.expanded;
+    },
+
+    expand() {
+        if (!this.isExpanded()) {
```

Comment:
Actually, we might need an isCollapsed method since this can evaluate to true if expander is null.

Reply:
> We have been slowly changing these kind of methods to getters mainly for code readability. This is our decision as owners of this repo.

What does "code readability" mean for the owner of the repo?

For me it is "easily understand what the code does".  And `setters` and `getters` make a lot hard to understand what the code really does. See https://github.com/eBay/ebayui-core/pull/1785#issuecomment-1273979641

Reply:
I realized that the last comment is in the wrong section.

Reply:
You are right that there is a problem here. 

Adding `isCollapsed` works to prevent accessing `this.expander` when it is `null`.